ansible/roles/kubernetes/tasks/gen_certs.yml

```diff
@@ -37,6 +37,7 @@
     HTTP_PROXY: "{{ http_proxy|default('') }}"
     HTTPS_PROXY: "{{ https_proxy|default('') }}"
     CLUSTER_HOSTNAME: "{{ master_cluster_hostname |default('') }}"
+    CLUSTER_PUBLIC_HOSTNAME: "{{ master_cluster_public_hostname |default('') }}"
```

Comment:
Could you remove the spaces between `|default` and the variable.
(On line 39 and 40)